content/reference/v1beta1/helmchart.md

```diff
@@ -25,6 +25,9 @@ spec:
 
   exclude: "repl{{ ConfigOptionEquals `include_chart` `include_chart_no`}}"
 
+  # helmVersion identifies the Helm Version used to render the Chart. Default is v2.
+  helmVersion: v2
```

Comment:
it says `apiVersion` below -- which one is correct?

Reply:
good catch. fixed